packages/create-contentful-app/template/src/components/ConfigScreen.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
+import { Heading, Form, Workbench, Paragraph } from '@contentful/forma-36-react-components';
+import { css } from 'emotion';
+
+export interface AppInstallationParameters {}
```

Comment:
Maybe add a comment why we have here an empty interface

Reply:
I'm actually not 100% sure if we should use parameters in the example. I think it's a remnant of these files being copied from the default value backend app. If you can think of a good example of using parameters for a beginner app - I'm happy to add it here